src/brackets.js

```diff
@@ -437,7 +438,9 @@ define(function (require, exports, module) {
         }
 
         if (brackets.platform === "win" && !brackets.inBrowser) {
-            PreferencesManager.definePreference("_windowsScrollFix", "boolean", true).on("change", enableOrDisableWinScrollFix);
+            PreferencesManager.definePreference("_windowsScrollFix", "boolean", true, {
```

Comment:
Exclude certain preferences from code hints:  will add an issue to track this after this is merged.